test/e2e/autoscaling/dns_autoscaling.go

```diff
@@ -166,8 +166,7 @@ var _ = SIGDescribe("DNS horizontal autoscaling", func() {
 		framework.ExpectNoError(err)
 	})
 
-	// TODO: Get rid of [DisabledForLargeClusters] tag when issue #55779 is fixed.
-	ginkgo.It("[DisabledForLargeClusters] kube-dns-autoscaler should scale kube-dns pods in both nonfaulty and faulty scenarios", func() {
+	ginkgo.It("kube-dns-autoscaler should scale kube-dns pods in both nonfaulty and faulty scenarios", func() {
```

Comment:
Since https://github.com/kubernetes/kubernetes/issues/55779 is fixed, this is now safe to enable?

Reply:
I think the fact that it's fixed doesn't mean anything.
But with endpoint-slices and couple other improvements that happened in the meantime it should be safe to enable it after those 3 years...